software/distro/setup/planktoscope-app-env/setup.sh

```diff
@@ -45,6 +45,14 @@ else
   panic "$description"
 fi
 
+description="set up /home/pi/PlanktoScope"
+report_starting "$description"
+if "$build_scripts_root/PlanktoScope/install.sh"; then
```

Comment:
If your'e stuck on https://github.com/PlanktoScope/PlanktoScope/actions/runs/14386498419/job/40343129723#step:15:3108 ("/run/os-setup/software/distro/setup/planktoscope-app-env/setup.sh: line 50: /run/os-setup/software/distro/setup/planktoscope-app-env/PlanktoScope/install.sh: Permission denied"), probably the cause of that error is that `.../PlanktoScope/install.sh`'s file permissions are currently `.rw-r--r--` instead of `.rwxr-xr-x`.